tests/Integration/Database/EloquentWhereHasMorphTest.php

```diff
@@ -135,6 +135,10 @@ public function testWhereHasMorphWithOwnerKey()
             $table->string('slug')->nullable();
         });
 
+        Schema::table('comments', function (Blueprint $table) {
+            $table->dropIndex('comments_commentable_type_commentable_id_index');
+        });
```

Comment:
MSSQL was nagging about the index when changing the related column so let's drop it explicitly.